src/MiniCover.HitServices/TestMethodUtils.cs

```diff
@@ -31,10 +44,15 @@ private static bool HasPdb(MethodBase methodBase)
             lock (Lock)
             {
                 if (AssemblyHasPdbCache.TryGetValue(location, out var hasPdb)) return hasPdb;
-                hasPdb = File.Exists(Path.ChangeExtension(location, ".pdb"));
+                hasPdb = IsNotATestFrameworkAssembly(methodBase.DeclaringType.Assembly) && File.Exists(Path.ChangeExtension(location, ".pdb"));
```

Comment:
Maybe rename hasPdb to isUserAssembly. Or something different.

Reply:
Yes I was thinking the same